profiles/permissions.restrictive

```diff
@@ -4,31 +4,24 @@
 # Copyright (c) 2019 SUSE Software Solutions Germany GmbH, Germany.
 #
 # Author: Roman Drahtmueller <draht@suse.de>, 2001
+# Author: Matthias Gerstner <matthias.gerstner@suse.de>, 2019
 #
-# 
-# See /etc/permissions for general hints on how to use this file.
+# See /etc/permissions and `man 5 permissions` for general hints on how to use
+# this file.
 #
-# /etc/permissions.paranoid is NOT designed to be used in a single-user as
-# well as a multi-user installation, be it networked or not.
+# This permissions profile has most sgid and suid bits cleared. As a result
+# parts of the system may become unusable for non-privileged users. In
+# addition, some of the configuration files are not readable for world any
+# more. This profile is considered to be used for very security sensitive use
+# cases where ease of use is not a requirement.
 #
```

Comment:
-> " In addition, some configuration files are not readable by every user anymore. You should consider this profile for very security sensitive use cases where ease of use is not a requirement."

The German in me also wants to add dashes in security-sensitive, use-cases, ease-of-use. But I'm not sure whether dashes actually belong there.

Reply:
I don't think dashes are necessary here. Even thought I'm German.